code/modules/unit_tests/breath.dm

```diff
@@ -53,9 +74,7 @@
 	TEST_ASSERT(!lab_rat.internal, "Plasmaman toggle_internals() failed to toggle internals")
 
 /// Tests to make sure ashwalkers can breath from the lavaland air
-/datum/unit_test/breath_sanity_ashwalker
```

Comment:
Keep this

Reply:
Rather, for everything here, keep the documentation for the individual unit test, NOT `Run()`